crates/transaction-pool/src/maintain.rs

```diff
@@ -83,7 +85,12 @@ pub async fn maintain_transaction_pool<Client, P, St, Tasks>(
         let info = BlockInfo {
             last_seen_block_hash: latest.hash,
             last_seen_block_number: latest.number,
-            pending_basefee: latest.next_block_base_fee().unwrap_or_default(),
+            pending_basefee: latest
+                .next_block_base_fee(
+                    client.chain_spec().elasticity_multiplier,
+                    client.chain_spec().base_fee_change_denominator,
```

Comment:
same re `Arc` comment above for 3 occurrences in this file

Reply:
done